src/shared/testbase/testsuitebase.go

```diff
@@ -61,7 +62,7 @@ func (s *ControllerManagerTestSuiteBase) BeforeTest(_, testName string) {
 		s.Require().NoError(err)
 	}()
 
-	s.TestNamespace = strings.ToLower(fmt.Sprintf("%s-%s", testName, time.Now().Format("20060102150405")))
+	s.TestNamespace = strings.ToLower(fmt.Sprintf("%s-%s", testName, time.Now().Format("20060102150405")))[:maxNamespaceLength]
```

Comment:
Bugfix - if test name was too long, then the resulting namespace was >63 chars, which made the test fail to initialize.